### ISSUE TITLE
Why use Trinity instead of Vicuna for asembly?

### ISSUE THREAD

**Issue by biocyberman** [broadinstitute/viral-ngs#572]
Just curious about this. Vicuna is a virus genome aware assembler, while Trinity - as described by its developers - is an RNA assembler. What is the rationale of this choice?

**Comment by dpark01** [comment]
We have tried both in the past with similar results in most scenarios, but with a slight edge to Trinity (specifically, an older version of it) in very low abundance situations (where very small numbers of reads exist, and only a couple short contigs can possibly be made). It wouldn't be a bad idea to modularize the *de novo* assembly step in a way that offered multiple assembler choices -- this has been something I've been thinking about for a while, but simply hasn't been pressing enough for us to commit time to.

But if you have some example read data where some other assemblers are performing much better than Trinity, I would love to hear more. Particularly, which assemblers you might be most interested in (SOAPdenovo? ABySS? Vicuna?)

Also, similar to V-Phaser, we no longer have in-house expertise on the internals of Vicuna, so in many ways, that assembler is no longer being actively developed and supported.

**Comment by biocyberman** [comment]
I did not use SOAPdenovo, and a little bit of ABySS before coming to Spades http://cab.spbu.ru/software/spades/.
I used SPades for assembling of some fungal genomes and it works great! It is written in python/Cython. Spades itself is a pipeline of error correction, and _de novo_ assembly.  

On the other hands, two recent publications about virus genome pipelines use ABySS and Oases : 
https://github.com/chiulab/surpi
https://github.com/keylabivdc/VIP

Maturity of the code and coding style in these pipelines turned me off, especially the VIP. So I came to this viral-ngs and feel more comfortable and inspired to work with. So I will first try to finish running with Trinity (viral-ngs) and see what next.
